velox/functions/prestosql/CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ add_library(
   MapEntries.cpp
   MapFromEntries.cpp
   MapKeysAndValues.cpp
+  MapTopNKeys.h
+  MapTopNKeysTest.cpp
```

Comment:
This belongs in the tests subdirectory: velox/functions/prestosql/tests/CMakeLists.txt